File: Plugins/Aspose_Email_Java_for_PHP/src/aspose/email/ProgrammingEmail/GetEmailInfo.php
<?php
namespace Aspose\Email\ProgrammingEmail;

use com\aspose\email\MessageFormat as MessageFormat;
use com\aspose\email\MailMessage as MailMessage;

class GetEmailInfo{

    public static function run($dataDir=null){

        # Create MailMessage instance by loading an Eml file
        $message_format = new MessageFormat();
        $mailMessage=new MailMessage();
        $message = $mailMessage->load($dataDir . "Message.eml");

        print "From: " . (string)$message->getFrom();

        print "To: " . (string)$message->getTo();

        print "Subject: " . (string)$message->getSubject();

        print "HtmlBody: " . (string)$message->getHtmlBody();

        print "TextBody: " . (string)$message->getTextBody();
    }

}
?>

File: Plugins/Aspose_Email_Java_for_Jython/asposeemail/ProgrammingEmail/GetEmailInfo.py
from asposeemail import Settings
from com.aspose.email import MailMessage
from com.aspose.email import MessageFormat

class GetEmailInfo:

    def __init__(self):
                        
        dataDir = Settings.dataDir + 'ProgrammingEmail/GetEmailInfo/'
        
        # Create MailMessage instance by loading an Eml file
        message_format = MessageFormat
        mailMessage=MailMessage()
        message = mailMessage.load(dataDir + "Message.eml")

        print "From: " 
        print message.getFrom()

        print "To: " 
        print message.getTo()

        print "Subject: " 
        print message.getSubject()

        print "HtmlBody: " 
        print message.getHtmlBody()

        print "TextBody: " 
        print message.getTextBody()
        
 
if __name__ == '__main__':        
    GetEmailInfo()